instances/treasury-factory.near/widget/components/create-treasury/CreateSputnikAccountStep.jsx

```diff
@@ -27,7 +27,7 @@ return (
           alertMsg,
           setAlertMsg,
           defaultValue: formFields.sputnikAccountName,
-          postfix: ".sputnik-dao.near",
```

Comment:
This postfix will still be `.sputnik-dao.near`. It is the web4 contract that will have its postfix changed from `.treasury-factory.near` to `.near`